RELEASE.md

```diff
@@ -1,3 +1,6 @@
+# Release 0.3.1
+- This release explicitly adds versions for dependencies, rather than leaving it up to the cellmlmanip and Jinja2 packages. Versions are semi-strict allowing for minor updates (which should not breat compatibility)
```

Comment:
```suggestion
- This release explicitly adds versions for dependencies, rather than leaving it up to the cellmlmanip and Jinja2 packages. Versions are semi-strict allowing for minor updates (which should not break compatibility).
```